test/index.js

```diff
@@ -90,4 +90,19 @@ describe('stylelint-webpack-plugin', function () {
         expect(stats.compilation.warnings).to.have.length(1);
       });
   });
+
+  it('works without StyleLintPlugin configuration but posts warnign .stylelintrc file not found', function () {
+    var config = {
+      context: './test/fixtures/test9',
+      entry: './index',
+      plugins: [
+        new StyleLintPlugin()
+      ]
+    };
+
+    return pack(assign({}, baseConfig, config))
+      .then(function (stats) {
+        expect(stats.compilation.errors).to.have.length(0);
```

Comment:
Could you add the assertion for the `stats.compilation.warnings` as well (since the test title implies it)?